risc0/binfmt/src/lib.rs

```diff
@@ -29,9 +29,6 @@ mod image;
 mod image2;
```

Comment:
should we drop the image mod and rename image2 -> image?

Reply:
also, I'm fine doing this on a later pass

Reply:
I have a feeling this would be a breaking change.

Reply:
yeah, maybe we could consider bumping binfmt to 2.0?